packages/cluster-ui/src/statementsTable/statementsTable.tsx

```diff
@@ -129,6 +143,7 @@ export interface AggregateStatistics {
   drawer?: boolean;
   firstCellBordered?: boolean;
   diagnosticsReports?: cockroach.server.serverpb.IStatementDiagnosticsReport[];
+  totalWorkload?: Long;
```

Comment:
nit: I think some comments here on `totalWorkload` and elsewhere would be useful

Reply:
done